pkg/node/manager.go

```diff
@@ -32,20 +32,17 @@ import (
 
 var log = logging.DefaultLogger
 
-// RouteType represents the route type to be configured when adding the node
-// routes
-type RouteType int
-
 const (
-	// TunnelRoute is the route type to set up the BPF tunnel maps
-	TunnelRoute RouteType = 1 << iota
-	// DirectRoute is the route type to set up the L3 route using iproute
-	DirectRoute
+	defaultClusterName = "default"
+
+	noWarningOnError = false
```

Comment:
Perhaps a comment explaining where these are intended to be used. They're a little weird in isolation.